code/modules/clothing/rogueclothes/rings.dm

```diff
@@ -18,11 +18,54 @@
 	icon_state = "ring_s"
 	sellprice = 33
 
+/obj/item/clothing/neck/roguetown/psicross/ring   /////////////////////// cast focus ring for acolytes and shit.
+	name = "psi ring"
+	desc = "a blessed psi ring, for the religious type"
+	w_class = WEIGHT_CLASS_TINY
+	icon = 'icons/roguetown/clothing/rings.dmi'
+	mob_overlay_icon = 'icons/roguetown/clothing/onmob/rings.dmi'
+	sleeved = 'icons/roguetown/clothing/onmob/rings.dmi'
+	sleevetype = "shirt"
+	icon_state = "castring"
+	slot_flags = ITEM_SLOT_RING
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+	anvilrepair = /datum/skill/craft/blacksmithing
+
```

Comment:
Why is it a subtype of the necklace instead of being a normal ring?

Reply:
you can just make it a ring and add it to the spells `req_items` list. it's cleaner and not "a shit ton of extra work"